projects/storefrontapp-e2e-cypress/cypress/e2e/b2b/regression/quote/quote.e2e-flaky.cy.ts

```diff
@@ -25,44 +35,186 @@ context('Quote', () => {
       },
     };
     cy.cxConfig(globalMessageSettings);
+    quote.log('visit default url', 'beforeEach');
```

Comment:
```suggestion
```
i would delete it. not much value of this log

Reply:
done